Feliz.UseMediaQuery/Feliz.UseMediaQuery.fsproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <PackageVersion>0.1</PackageVersion>
+        <Authors>Jonas Bösch, Zaid Ajaj</Authors>
+        <Description>useMediaQuery hooks to build responsive websites </Description>
+        <PackageLicenseUrl>https://github.com/Zaid-Ajaj/Feliz/blob/master/LICENSE</PackageLicenseUrl>
+        <RepositoryUrl>https://github.com/Zaid-Ajaj/Feliz</RepositoryUrl>
+        <PackageTags>fsharp;fable;react;html;feliz</PackageTags>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Content Include="*.fsproj; *.fs; *.js;" PackagePath="fable\" />
+        <Compile Include="UseMediaQuery.fs" />
+        <Content Update="Breakpoints.fs">
+          <PackagePath>fable\</PackagePath>
+        </Content>
```

Comment:
I am not sure about what this part is doing here 🤔 shouldn't this just be this?
```xml
<Compile Include="Breakpoints.fs" />
```
Actually I don't even see the file there, maybe added by mistake?

Reply:
Will fix.